src/helpers/LightProbeHelper.js

```diff
@@ -0,0 +1,152 @@
+import { Mesh } from '../objects/Mesh.js';
+import { ShaderMaterial } from '../materials/ShaderMaterial.js';
+import { SphereBufferGeometry } from '../geometries/SphereGeometry.js';
+
+/**
+ * @author WestLangley / http://github.com/WestLangley
+ */
+
+function LightProbeHelper( lightProbe, size ) {
+
+	this.lightProbe = lightProbe;
+
+	this.size = size;
+
+	var defines = {};
+	defines[ 'GAMMA_OUTPUT' ] = "";
+
+	// material
+	var material = new ShaderMaterial( {
```

Comment:
In the meanwhile, it would be preferable to have `LightProbeHelper` without a dependency to `ShaderMaterial` so the helper is directly usable in `WebGPURenderer`.

Instead of directly evaluate the probe in a shader, how about converting the probe to an (irradiance) environment map and apply that to the sphere. The texture itself could be generated via JS as a`CanvasTexture`.

Since instances of `LightProbe` should present baked lighting, updating the texture does not have to be a fast operation (so generating it in JS is fine).